lib/trento.pm

```diff
@@ -435,10 +435,10 @@ Deploy a SAP Landscape using a previously configured qe-sap-deployment
 =cut
 
 sub cluster_deploy {
-    my $ret = qesap_execute(cmd => 'terraform', verbose => 1, timeout => bmwqemu::scale_timeout(1800));
-    die "'qesap.py terraform' return: $ret" if ($ret);
-    $ret = qesap_execute(cmd => 'ansible', verbose => 1, timeout => bmwqemu::scale_timeout(3600));
-    die "'qesap.py ansible' return: $ret" if ($ret);
+    my @ret = qesap_execute(cmd => 'terraform', verbose => 1, timeout => bmwqemu::scale_timeout(1800));
```

Comment:
Same recommendation. This could have been:
```
my ($ret) = qesap_execute(cmd => 'terraform', verbose => 1, timeout => bmwqemu::scale_timeout(1800));
```
And L439 would not have required any change.